app/cdash/app/Model/CoverageFile2User.php

```diff
@@ -289,31 +291,18 @@ public function GetCoverageFileId($buildid): int|false
     /**
      * Get the list of authors for the project
      *
-     * @return array<int>|false
+     * @return array<int>
      */
-    public function GetUsersFromProject(): array|false
+    public function GetUsersFromProject(): array
```

Comment:
This function is only used in one location, and this change makes the function rather trivial.  The query doesn't even really relate to this model anyway...

Would it make sense to just eliminate this function and inline the contents?